### PR TITLE
Update Network types locales + add help text on form

### PR DESCRIPTION
**Pull request by B-Rass**

### CODE REVIEW

**Comment by nicolas-brousse** [comment]
@B-Rass did you missed something here? I can't really review this, at least now.

**Comment by B-Rass** [comment]
> @B-Rass did you missed something here? I can't really review this, at least now.

apparently ... ^^'

**Comment by B-Rass** [comment]
> @B-Rass did you missed something here? I can't really review this, at least now.



> > @B-Rass did you missed something here? I can't really review this, at least now.
> 
> apparently ... ^^'

@nicolas-brousse fixed

File: app/views/modeles/show.html.erb [hunk 1]
```diff
@@ -16,9 +16,13 @@
...
 
-        <dt class="pb-2"><%= Modele.human_attribute_name(:network_types) %> :</dt>
+        <dt class="pb-2"><%= Modele.human_attribute_name(:network_types) %></dt>
         <dd class="mb-0 pb-2 ps-3">
-          <%= @modele.network_types.map { |type| Modele.human_attribute_name("network_types.#{type}") }.join(", ") %>
+          <% if network_types = @modele.network_types.presence %>
+            <%= network_types.map { |type| Modele.human_attribute_name("network_types.#{type}") }.join(", ") %>
+          <% else %>
+            <%= Modele.human_attribute_name("network_types.blank")  %>
+          <% end %>
```
Comment (nicolas-brousse) [inline app/views/modeles/show.html.erb]: Could be done in a decorator helper?

File: app/views/servers/show.html.erb [hunk 2]
```diff
@@ -200,7 +200,11 @@
 
           <dt class="pb-2"><%= Server.human_attribute_name(:network_types) %></dt>
           <dd class="mb-0 pb-2 ps-3">
-            <%= @server.network_types.map { |type| Modele.human_attribute_name("network_types.#{type}") }.join(", ") %>
+            <% if network_types = @server.network_types.presence %>
+              <%= network_types.map { |type| Modele.human_attribute_name("network_types.#{type}") }.join(", ") %>
+            <% else %>
+              <%= Modele.human_attribute_name("network_types.blank")  %>
+            <% end %>
```
Comment (nicolas-brousse) [inline app/views/servers/show.html.erb]: Could be done in a decorator helper?

**Review by nicolas-brousse — APPROVED** [review]

**Pull request merged**